firebase-sessions/test-app/src/main/kotlin/com/google/firebase/testing/sessions/MainActivity.kt

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.firebase.testing.sessions
+
+import android.annotation.SuppressLint
+import android.os.Bundle
+import android.widget.TextView
+import androidx.appcompat.app.AppCompatActivity
+import com.google.firebase.sessions.FirebaseSessions
+
+class MainActivity : AppCompatActivity() {
+  @SuppressLint("DiscouragedApi")
+  override fun onCreate(savedInstanceState: Bundle?) {
+    super.onCreate(savedInstanceState)
+    setContentView(R.layout.activity_main)
+
+    findViewById<TextView>(R.id.greeting_text).text = FirebaseSessions.instance.greeting()
```

Comment:
Can we have some other random text here? Is the intent to refer to FIrebaseSessions to verify if the integration is working?

My concern with this is that some auto initializing FirebaseSessions functionality could not be verified.

Reply:
Yeah this was for local testing. Not calling it explicitly makes sense for when this is a CI test, I removed it. I have verified the auto initializing locally by seeing the message in logcat.